### PR TITLE
[frontend] fix import openpgp manager

### PR DESCRIPTION
**Pull request by iamdey**
refactor double declaration of `generateKey`

### CODE REVIEW

**Comment by iamdey** [comment]
not fully verified yet

**Comment by iamdey** [comment]
should be fine now

File: src/frontend/web_application/src/components/InputFileGroup/index.jsx [hunk 1]
```diff
@@ -116,8 +117,8 @@ class InputFileGroup extends Component {
         {descr && <p>{descr}</p>}
 
         {this.state.files.length > 0 ? (
-          this.state.files.map((file) => (
-            <File file={file} onRemove={this.resetForm} />
+          this.state.files.map((file, i) => (
+            <File key={i} file={file} onRemove={this.resetForm} />
```
Comment (MonsieurLanza) [inline src/frontend/web_application/src/components/InputFileGroup/index.jsx:121]: No array index in key… Mais que fait `eslint` ? :p

File: src/frontend/web_application/src/scenes/UserSecurity/components/OpenPGPKey/index.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                </Button>
+              )}
+            </div>
+          </>
+        )}
+      </div>
+
+      {keyDetails && (
+        <div className="m-openpgp-key__summary">
+          <span>{keyDetails?.userIds}</span>
```
Comment (MonsieurLanza) [inline src/frontend/web_application/src/scenes/UserSecurity/components/OpenPGPKey/index.tsx:153]: `userIds` are thrown as is, without separators, which is difficult to read. We may want to dive in a little bit to see how to display this in a better way.

File: src/frontend/web_application/src/components/InputFileGroup/index.jsx (hunk 1 above)
Comment (iamdey) [reply src/frontend/web_application/src/components/InputFileGroup/index.jsx:121]: at least index is better than nothing :)

File: src/frontend/web_application/src/scenes/UserSecurity/components/OpenPGPKey/index.tsx (hunk 2 above)
Comment (iamdey) [reply src/frontend/web_application/src/scenes/UserSecurity/components/OpenPGPKey/index.tsx]: fixed

**Pull request merged**